incubator/virtualcluster/cmd/vn-agent/app/server.go

```diff
@@ -136,5 +137,10 @@ func Run(c *config.Config, serverOption *options.ServerOption, stopCh <-chan str
 		klog.Errorf("server listen error %v", err)
 	}
 
+	// start a health http server.
+	mux := http.NewServeMux()
+	healthz.InstallHandler(mux)
+	klog.Fatal(http.ListenAndServe(":8080", mux))
```

Comment:
Likewise, this needs to be run in a goroutine.

Reply:
Put it in goroutine